src/transactionsTable/transactionsTable.tsx

```diff
@@ -103,9 +107,11 @@ export const TransactionsTable: React.FC<TransactionsTable> = props => {
   ];
 
   return (
-    <TransactionsSortedTable
+    <SortedTable
       data={data}
-      columns={columns as any}
+      // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+      // @ts-ignore
+      columns={columns}
```

Comment:
To avoid lots of type incompatibilities with defining columns - it is needed to refactor `makeBarChart` function to be generic. It will allow us to reuse it across Statements and Transactions.